src/components/events/CampusSafetyIncidentReporterWidget.tsx

```
@@ -0,0 +1,317 @@
+  eventId = "evt-concert-2026",
+  eventTitle = "Annual Campus Spring Music Concert",
+  initialReport = null,
+  onReportSubmitted,
+  className,
+}) => {
+  const [activeReport, setActiveReport] = useState<IncidentReportPayload | null>(initialReport);
+  const [showTriageModal, setShowTriageModal] = useState<boolean>(false);
+  const [selectedCategory, setSelectedCategory] = useState<IncidentCategory>("medical_emergency");
+  const [locationLabel, setLocationLabel] = useState<string>("Near North Stage");
+  const [description, setDescription] = useState<string>("Student collapsed due to heat exhaustion.");
+  const [userCoords, setUserCoords] = useState<{ lat: number; lng: number }>({ lat: 37.7749, lng: -122.4194 });
```

ESLint

[error] Replace `·lat:·37.7749,·lng:·-122.4194` with `⏎····lat:·37.7749,⏎····lng:·-122.4194,⏎·`

(prettier/prettier)